mutiply_code_test.py

```diff
@@ -0,0 +1,6 @@
+from mutiply_code import mutiply
+
+def test_code():
```

Comment:
By convention the test method is names after the method it is testing so this would be `test_multiply`

Reply:
fixed it